routes/discord.js

```diff
@@ -18,4 +23,54 @@ router.post('/generate-invite', verifyRole(ROLE_NAMES.member), async (req, res)
   } 
 });
 
+router.get('/auth/discord', (req, res) => {
+  res.redirect(`https://discord.com/api/oauth2/authorize?client_id=${discordAuth.discordClientId}
+    &redirect_uri=${encodeURIComponent(discordAuth.discordRedirectUrl)}&response_type=code&scope=identify%20email`);
+});
+
+router.get('/auth/discord/callback', async (req, res) => {
+  const { code } = req.query;
+  
+  if (!code) return  respondWithError(res, 400, 'no code provided');
```

Comment:
as discussed yesterday, let's move this logic into /users/login endpoint.
things to consider:
- rename 'token' value to be better-descriptive, i.e. 'firebaseToken'.
- add check for discord code, also name it accordingly
- if no suitable tokens found in headers -> return 401
- if some token/code found, process according to the service the token comes from
- a suggestion: add some authentication service that would perform the checks below